lib/vsc/mympirun/rm/pbs.py

```diff
@@ -37,6 +37,8 @@ class PBS(Sched):
     SCHED_ENVIRON_ID = 'PBS_JOBID'
     SCHED_ENVIRON_NODEFILE = 'PBS_NODEFILE'
 
+    # pbsssh is only used if native support for pbsdsh is not supported in the MPI library;
+    # e.g. Intel MPI v5.x and newer supports using pbsdsh as launcher natively, no need for pbsssh wrapper
```

Comment:
follow-up in https://github.com/hpcugent/vsc-mympirun/issues/112, but #113 suggest we're already doing the right thing (we should just add a test for it)